source/adiabatic_conditions/compute_profile.cc

```diff
@@ -79,8 +93,16 @@ namespace aspect
         {
           if (i==0)
             {
-              pressures[i] = this->get_surface_pressure();
-              temperatures[i] = this->get_adiabatic_surface_temperature();
+              if (!use_surface_condition_function)
+                {
+                  pressures[0] = this->get_surface_pressure();
+                  temperatures[0] = this->get_adiabatic_surface_temperature();
+                }
+              else
+                {
+                  pressures[0] = surface_condition_function.value(Point<1>(),0);
```

Comment:
why do you use the default constructor instead of ``Point<1>(0.0)``? This is supposed to be depth=0 or not?

Reply:
Aren't the two equivalent? Anyway, I changed it to make it more explicit.